src/test/java/net/sf/jabref/logic/exporter/MSBibExportFormatTestFiles.java

```diff
@@ -70,15 +74,17 @@ public final void testPerformExport() throws IOException, SaveException {
         String xmlFileName = filename.replace(".bib", ".xml");
         Path importFile = resourceDir.resolve(filename);
         String tempFilename = tempFile.getCanonicalPath();
+
         List<BibEntry> entries = testImporter.importDatabase(importFile, StandardCharsets.UTF_8).getDatabase()
                 .getEntries();
 
         msBibExportFormat.performExport(databaseContext, tempFile.getPath(), charset, entries);
 
-        List<String> expected = Files.readAllLines(resourceDir.resolve(xmlFileName));
-        List<String> exported = Files.readAllLines(Paths.get(tempFilename));
-        Collections.sort(expected);
-        Collections.sort(exported);
-        assertEquals(expected, exported);
+        Builder control = Input.from(Files.newInputStream(resourceDir.resolve(xmlFileName)));
+        Builder test = Input.from(Files.newInputStream(Paths.get(tempFilename)));
+
+        assertThat(test, CompareMatcher.isSimilarTo(control)
```

Comment:
@tobiasdiez  I tried a bit around and I could get it working with assertFalse. However, it has the disadvantage that it does not throw a ComparisonFailure exception which would allow you to see the details directly in the ide:
https://github.com/xmlunit/user-guide/wiki/CompareMatcher#throwcomparisonfailure